packages/kbn-test/src/functional_test_runner/cli.ts

```diff
@@ -141,6 +150,7 @@ export function runFtrCli() {
         --exclude-tag=tag  a tag to be excluded, pass multiple times for multiple tags
         --test-stats       print the number of tests (included and excluded) to STDERR
         --updateBaselines  replace baseline screenshots with whatever is generated from the test
+        --updateSnapshots  replace inline and file snapshots with whatever is generated from the test
```

Comment:
I definitely reached for the `-u` flag here, maybe it would be nice to have an `--update,-u` flag that sets both `updateBaselines` and `updateSnapshots`.

Reply:
done - I tried to add `-u` only where necessary, but let me know if I missed anything.